alpaca_trade_api/stream2.py

```diff
@@ -140,12 +141,15 @@ async def unsubscribe(self, channels):
             }))
 
     async def close(self):
-        if self._consume_task:
-            self._consume_task.cancel()
+        await self.cancel_task()
         if self._ws:
             await self._ws.close()
             self._ws = None
 
+    async def cancel_task(self):
```

Comment:
[minor] should be private: `_cancel_task()`

Reply:
same. used out side of the class.